[2024-12-13T15:33:59.820+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-12-13T15:34:00.123+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_data.insert_data.insert_data_student manual__2024-12-13T15:33:48.149184+00:00 [queued]>
[2024-12-13T15:34:00.134+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_data.insert_data.insert_data_student manual__2024-12-13T15:33:48.149184+00:00 [queued]>
[2024-12-13T15:34:00.134+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-12-13T15:34:00.150+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): insert_data.insert_data_student> on 2024-12-13 15:33:48.149184+00:00
[2024-12-13T15:34:00.169+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=390) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-13T15:34:00.171+0000] {standard_task_runner.py:63} INFO - Started process 392 to run task
[2024-12-13T15:34:00.172+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_data', 'insert_data.insert_data_student', 'manual__2024-12-13T15:33:48.149184+00:00', '--job-id', '89', '--raw', '--subdir', 'DAGS_FOLDER/etl_data.py', '--cfg-path', '/tmp/tmp1zwdv2h2']
[2024-12-13T15:34:00.178+0000] {standard_task_runner.py:91} INFO - Job 89: Subtask insert_data.insert_data_student
[2024-12-13T15:34:00.242+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_data.insert_data.insert_data_student manual__2024-12-13T15:33:48.149184+00:00 [running]> on host 1fe973f28a1c
[2024-12-13T15:34:00.390+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_data' AIRFLOW_CTX_TASK_ID='insert_data.insert_data_student' AIRFLOW_CTX_EXECUTION_DATE='2024-12-13T15:33:48.149184+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-13T15:33:48.149184+00:00'
[2024-12-13T15:34:00.394+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-12-13T15:34:00.754+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-12-13T15:34:00.756+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-12-13T15:34:00.776+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_data, task_id=insert_data.insert_data_student, run_id=manual__2024-12-13T15:33:48.149184+00:00, execution_date=20241213T153348, start_date=20241213T153400, end_date=20241213T153400
[2024-12-13T15:34:00.830+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-12-13T15:34:00.873+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-13T15:34:00.876+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
